### PR TITLE
add_english_description_to_service_translation

### PR DESCRIPTION
**Pull request by JJJazl**
1) added variable descriptionEng in ServiceTranslation, ServiceTranslationDto, EditServiceDto and GetServiceDto.
2) added new xml file in dao/src/main/resources/db/changelog/logs 
3) changed CreateServiceDtoMapper, SerivceTransaltionDtoMapper, ServiceTranslationDtoMapperTest 
4) changed ModelUtils

### CODE REVIEW

File: dao/src/main/resources/db/changelog/db.changelog-master.xml [hunk 1]
```diff
@@ -146,5 +146,6 @@
     <include file="db/changelog/logs/ch-insert-into-order-status-translations-Yashna.xml"/>
     <include file="db/changelog/logs/ch-drop-column-language_id-in-courier_and_service_translations-Hlazova.xml"/>
     <include file="db/changelog/logs/ch-drop-column-language_id-in-notification_templates-Hlazova.xml"/>
+    <include file="db/changelog/logs/ch-add-column-description_eng-Safarov.xml"/>
```
Comment (RomaMocherniuk) [inline dao/src/main/resources/db/changelog/db.changelog-master.xml:149]: do we need `-Safarov` in the name?

File: service/src/test/java/greencity/ModelUtils.java [hunk 2]
```diff
@@ -2253,6 +2253,7 @@ public static EditServiceDto getEditServiceDto() {
             .locationId(1L)
             .price(100)
             .description("test")
+            .descriptionEng("test")
```
Comment (RomaMocherniuk) [inline service/src/test/java/greencity/ModelUtils.java]: could you please use different values for `description` and `descriptionEng` to be sure everything works as expected?

File: service/src/main/java/greencity/mapping/service/CreateServiceDtoMapper.java [hunk 3]
```diff
@@ -16,7 +16,7 @@ public class CreateServiceDtoMapper extends AbstractConverter<Service, CreateSer
     protected CreateServiceDto convert(Service source) {
         List<ServiceTranslation> serviceTranslationList = source.getServiceTranslations();
         List<ServiceTranslationDto> dtos = serviceTranslationList.stream().map(
-            i -> new ServiceTranslationDto(i.getName(), i.getDescription(), i.getNameEng()))
+            i -> new ServiceTranslationDto(i.getName(), i.getNameEng(), i.getDescription(), i.getDescriptionEng()))
```
Comment (RomaMocherniuk) [inline service/src/main/java/greencity/mapping/service/CreateServiceDtoMapper.java]: I think it is better to change` i -> item`

Comment (RomaMocherniuk) [reply service/src/main/java/greencity/mapping/service/CreateServiceDtoMapper.java]: you can use `toList()` instead of `collect(Collectors.toList())`

**Review by RomaMocherniuk — APPROVED** [review]

**Review by OlegVat — APPROVED** [review]

**Review by innayashna — APPROVED** [review]

**Review by NKorzhik — APPROVED** [review]

**Pull request merged**